File: Introduction to Programming I/Python/ITP1_4_C.py
def calculate(a,b,op):
    if op == '+':
        return a + b
    elif op == '-':
        return a-b
    elif op == '*':
        return a*b
    elif op == '/':
        return a//b 
    else:
        return -1

while True:
    input_str = input().strip()
    if input_str == "?":
        break
    a,op,b = input_str.split()
    a,b = int(a), int(b)
    result = calculate(a,b,op)
    print(result)

File: Introduction to Programming I/Java/ITP1_4_C.java
import java.util.*;
public class ITP1_4_C {
    public static int calculate(int a, int b, char op){
        switch(op){
            case '+':
                return a + b;
            case '-':
                return a-b;
            case '*':
                return a*b;
            case '/':
                return a/b;
            default:
                return -1;
        }
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while(true){
            String inputStr = sc.nextLine().trim();
            if(inputStr.equals("?")){
                break;
            }
            String[] parts = inputStr.split("\\s");
            int a = Integer.parseInt(parts[0]);
            char op = parts[1].charAt(0);
            int b = Integer.parseInt(parts[2]);

            int res = calculate(a, b, op);
            System.out.println(res);
        }
        sc.close();
    }
}
